packages/harmony/src/components/layout/Paper/Paper.mdx

```diff
@@ -0,0 +1,79 @@
+import {
+  Canvas,
+  Meta,
+  Title,
+  Subtitle,
+  Description,
+  Primary,
+  Controls,
+  Story,
+  Stories
+} from '@storybook/blocks'
+import { Paper } from './Paper'
+import { Box, Flex } from '..'
+import * as PaperStories from './Paper.stories'
+import paperExampleGood from '../../../../.storybook/public/paperExampleGood.png'
+
+<Meta of={PaperStories} />
+
+<Title />
+
+- [Overview](#overview)
+- [Props](#props)
+- [Use cases and examples](#use-cases-and-examples)
+- [Do's and dont's](#dos-and-donts)
+
+## Overview
+
+<Subtitle />
+<Description />
+<Primary />
+
+## Props
+
+<Controls />
+
+## Use cases and examples
+
+- The paper component has pre-set properties for color, corner radius, shadow, border color, and spacing.
+- Use the paper component to quickly create containers and eliminate guesswork.
+- Ensure consistency by using the paper component to create containers that follow Harmony’s foundational properties.
+
+## Do's and don'ts
+
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        description:
+          'Use paper to create spin-off component in your local library.',
+        component: (
+          <Flex h='100%' w='100%' alignItems='center' justifyContent='center'>
+            <img src={paperExampleGood} />
+          </Flex>
+        )
+      },
+      negative: {
+        description:
+          'Don’t apply properties not built into the Paper component.',
```

Comment:
Nice maybe that answers my question about styled(Flex)

Reply:
Yeah I think the point here is to be restrictive, though I think we'll want a flex child 90% of the time so maybe worth adding but not advertising the props? Open to input on that.

Reply:
Yeah let's add flex and maybe direction? Just cause otherwise folks will have to add a flex inside every time which is maybe okay honestly

Reply:
You think we should do a subset of `FlexProps` then? Or should we just let you do anything Flex can do to it but restrict use of the specific props we override here?